### PR TITLE
PFHub Upload: CHiMaD3_MOOSE

### PR DESCRIPTION
**Pull request by staticmanapp**
This a new PFHub upload

To review:

- Wait for the tests to pass.

- Go to http://random-cat-792.surge.sh/simulations/display/?sim=CHiMaD3_MOOSE

| Field     | Content                                                                                         |
| --------- | ----------------------------------------------------------------------------------------------- |
| metadata  | [object Object]                                                                                 |
| benchmark | [object Object]                                                                                 |
| data      | [object Object],[object Object],[object Object],[object Object],[object Object],[object Object] |
| date      | 1530219641                                                                                      |

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/792)

### CODE REVIEW

File: _data/simulations/CHiMaD3_MOOSE/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
+_id: 5060d3a0-7b16-11e8-b5f4-6170e8cb26a5
+metadata:
+  author:
+    first: Dong-Uk
+    last: Kim
+    email: cmskdu@gmail.com
+    github_id: eastglow-zz
+  timestamp: '27 June, 2018'
+  summary: >-
+    FiPy implementation of benchmark 3a on a 960x960 grid. The shape of the
```
Comment (wd15) [inline _data/simulations/CHiMaD3_MOOSE/meta.yaml]: This claims that you are using FiPy, but you're also claim to use Moose. Can you edit this?

**Comment by eastglow-zz** [comment]
Hi, 
Thanks for reviewing.
I've resolved the issues by making another folk. I also created the pull request. Please see #793 .
Thank you.
-Dong-Uk

**Comment by wd15** [comment]
@eastglow-zz, please it over again and give your approval. Look at both

 - http://random-cat-792.surge.sh/simulations/display/?sim=CHiMaD3_MOOSE
 - http://random-cat-792.surge.sh/simulations/3a.1/

and ensure you're happy.

**Comment by eastglow-zz** [comment]
Hi Daniel, 
Thanks for working on my upload. But, I found a bug in my postprocess code evaluating the total free energy, and I'm fixing it now. And I'm also planning to rearrange my contour data so that it can look better on the line plot. 
Could you hold this for a while please?
Thanks,
-Dong-Uk

**Comment by wd15** [comment]
@eastglow-zz, no worries. Your results did look different from the two others. I'll leave this pull request open. Let me know if you'd like to close it and start over.

**Comment by wd15** [comment]
@eastglow-zz, this is on the main site now, https://pages.nist.gov/pfhub/simulations/3a.1/. Please make sure you're happy with it and also any suggestions for making the site better are also useful.

Thanks for the contribution!

**Pull request merged**